File: java/TwoSum.java
public class TwoSum{
    public int[] twoSum(int[] nums, int target) {
        int[] rs = new int[2];
        for (int i = 0; i < nums.length; i++) {
            for (int j = i + 1; j < nums.length; j++) {
                if (nums[i] + nums[j] == target){
                    rs[0] = i;
                    rs[1] = j;
                    break;
                } else {
                    continue;
                }
            }
        }
        return rs;
    }

    public static void main(String[] args) {
		int[] nums = new int[]{1, 2, 3, 4};
		int[] results = new int[2];
		TwoSum ts = new TwoSum();
		results = ts.twoSum(nums, 5);
		for (int i : results) {
			System.out.print(i + " ");
		}
	}
}

File: python/TwoSum.py
class Solution:
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        res = []
        for i, num1 in enumerate(nums):
            for j, num2 in enumerate(nums):
                if num1 + num2 == target and i != j:
                    res = [i, j]
                else:
                    continue
        return sorted(res)

def main():
    solution = Solution()
    nums = [2, 7, 11, 15]
    target = 13
    print(solution.twoSum(nums, target))

if __name__ == '__main__':
    main()